lib/engine/game/g_1822/game.rb

```diff
@@ -700,13 +700,16 @@ def discountable_trains_for(corporation)
         end
 
         def end_game!
+          finalize_end_game_values
+          super
+        end
+
+        def finalize_end_game_values
           company = company_by_id(self.class::COMPANY_OSTH)
-          if company && @tax_haven.value.positive?
-            # Make sure tax havens value is correct
-            company.value = @tax_haven.value
-          end
+          return unless company && @tax_haven.value.positive?
 
-          super
```

Comment:
super no longer needed for 22?

Reply:
It's a new method, there's nothing to super.  That -super was from the end_game! method which does have it.